src/commands/logs/functions.mts

```diff
@@ -0,0 +1,101 @@
+import { Argument, OptionValues } from 'commander'
+import inquirer from 'inquirer'
+
+import { chalk, log } from '../../utils/command-helpers.mjs'
+import { getWebSocket } from '../../utils/websockets/index.mjs'
+import BaseCommand from '../base-command.mjs'
+
+function getLog(logData: { level: string; message: string }) {
+  let logString = ''
+  switch (logData.level) {
```

Comment:
Similar to comment above regarding potential deploy context values, we can do something similar for the `level`